beacon-chain/flags/rpc.go

```diff
@@ -0,0 +1,63 @@
+package flags
+
+import "github.com/urfave/cli/v2"
+
+var (
+	// HTTPWeb3ProviderFlag provides an HTTP access endpoint to an ETH 1.0 RPC.
+	HTTPWeb3ProviderFlag = &cli.StringFlag{
+		Name:  "http-web3provider",
+		Usage: "A mainchain web3 provider string http endpoint",
+		Value: "",
+	}
+	// RPCHost defines the host on which the RPC server should listen.
+	RPCHost = &cli.StringFlag{
+		Name:  "rpc-host",
+		Usage: "Host on which the RPC server should listen",
+		Value: "127.0.0.1",
+	}
+	// RPCPort defines a beacon node RPC port to open.
+	RPCPort = &cli.IntFlag{
+		Name:  "rpc-port",
+		Usage: "RPC port exposed by a beacon node",
+		Value: 4000,
+	}
+	// CertFlag defines a flag for the node's TLS certificate.
+	CertFlag = &cli.StringFlag{
+		Name:  "tls-cert",
+		Usage: "Certificate for secure gRPC. Pass this and the tls-key flag in order to use gRPC securely.",
+	}
+	// KeyFlag defines a flag for the node's TLS key.
+	KeyFlag = &cli.StringFlag{
+		Name:  "tls-key",
+		Usage: "Key for secure gRPC. Pass this and the tls-cert flag in order to use gRPC securely.",
+	}
+	// DisableGRPCGateway for JSON-HTTP requests to the beacon node.
+	DisableGRPCGateway = &cli.BoolFlag{
+		Name:  "disable-grpc-gateway",
+		Usage: "Disable the gRPC gateway for JSON-HTTP requests",
+	}
+	// GRPCGatewayHost specifies a gRPC gateway host for Prysm.
+	GRPCGatewayHost = &cli.StringFlag{
+		Name:  "grpc-gateway-host",
+		Usage: "The host on which the gateway server runs on",
+		Value: "127.0.0.1",
+	}
+	// GRPCGatewayPort enables a gRPC gateway to be exposed for Prysm.
+	GRPCGatewayPort = &cli.IntFlag{
+		Name:  "grpc-gateway-port",
+		Usage: "Enable gRPC gateway for JSON requests",
+		Value: 3500,
+	}
+	// GPRCGatewayCorsDomain serves preflight requests when serving gRPC JSON gateway.
+	GPRCGatewayCorsDomain = &cli.StringFlag{
+		Name: "grpc-gateway-corsdomain",
+		Usage: "Comma separated list of domains from which to accept cross origin requests " +
+			"(browser enforced). This flag has no effect if not used with --grpc-gateway-port.",
```

Comment:
This flag definitely has effect if used without changing --grpc-gateway-port.

```suggestion
			"(browser enforced).",
```